### PR TITLE
Improve wording of abandoning message

### PR DESCRIPTION
**Pull request by iHiD**
Fixes https://github.com/exercism/exercism/issues/4604

### CODE REVIEW

File: test/services/abandon_solution_mentorship_test.rb [hunk 1]
```diff
@@ -17,7 +17,7 @@ class AbandonSolutionMentorshipTest < ActiveSupport::TestCase
 
     # Check discussion post has been created
     content = {'i18n_message' => 'system_messages.mentor_timed_out', 'handle' => mentor.handle}
-    html = ParseMarkdown.("Mentor 'freddie' has being removed from this conversation due to inactivity. This solution will be placed into the queue for a new mentor to review.")
+    html = ParseMarkdown.("Mentor 'freddie' been being removed from this conversation due to inactivity. This solution has been resubmitted to the queue for a new mentor to review.")
```
Comment (F3PiX) [inline test/services/abandon_solution_mentorship_test.rb]: s/been being/had been

Comment (BethanyG) [reply test/services/abandon_solution_mentorship_test.rb]:
```suggestion
    html = ParseMarkdown.("Mentor 'freddie' has been removed from this conversation due to inactivity. This solution has been resubmitted to the queue for a new mentor to review.")
```

Comment (iHiD) [reply test/services/abandon_solution_mentorship_test.rb]: @BethanyG @F3PiX Thanks both. It's taken me three attempts to do this! More coffee needed!

Comment (F3PiX) [reply test/services/abandon_solution_mentorship_test.rb]: 🥳

**Pull request merged**